src/popup/router/components/TotalWalletAmount.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="total-amount">
+    <span>{{ totalAmount }}</span>
+    <span class="label">
+      {{ $t('total') }}
+    </span>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+
+export default {
+  subscriptions() {
+    return pick(this.$store.state.observables, ['balances']);
+  },
+  computed: {
+    ...mapGetters(['convertToCurrencyFormatted', 'accounts']),
```

Comment:
```suggestion
    ...mapGetters(['convertToCurrencyFormatted']),
```
`accounts` is not used in this component.